Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
```

Comment:
Not sure why we need pip, as we don't use python anywhere in the codebase.

Reply:
this is just for M1 & M2! 
`npm install`  fails without python 3.9
its just a convenient way to switch your python env with one command `pipenv shell`
if npm install works - you don't have to be use it at all